post-install.yml

```diff
@@ -122,6 +122,8 @@
       git:
         repo: https://github.com/tsuru/postgres-api
         dest: /tmp/postgresapi
+        version: "f25564ed29fc204d0207a54a28011e33789f89ca"
+        # version above is a merge from our PR enabling authentication
```

Comment:
Can you omit this comment? I suspect we'll forget to change it the next time we bump the version and it will no longer be relevant. The commit message already describes why we choose this version :+1: